python/ray/data/_internal/datasource/parquet_datasource.py

```diff
@@ -633,3 +650,17 @@ def sample_fragments(
     sample_bar.close()
 
     return sample_infos
+
+
+def _add_partitions_to_table(table, partitions):
```

Comment:
Please annotate types

Reply:
Not addressed

Reply:
Fixed in 4f15d25c99cae192fe95a37cf196b9253ceb47ed